classes/fields/file.php

```diff
@@ -711,6 +711,8 @@ public function data_image_sizes( $name = null, $value = null, $options = null,
 			$data[ $image_size ] = ucwords( str_replace( '-', ' ', $image_size ) );
 		}
 
+		$data[ 'full' ] = __( 'Full Size' ); // Translated by WordPress core.
```

Comment:
:no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: dc7bacb5ad727097d207a11573789bb9</sub>